### PR TITLE
chore(core): remove meson warnings for wasm builds

### PR DESCRIPTION
**Pull request by ermshiperete**
Replace deprecated function calls. Also add `strip` command to wasm build defaults.

@keymanapp-test-bot skip

### CODE REVIEW

File: core/tests/unit/kmx/meson.build [hunk 1]
```diff
@@ -139,7 +139,10 @@ foreach kbd : tests
   cfg.set('CONTENT', content)
 
   kbd_kmn = configure_file(
-    copy: true,
+    # not really a file that needs to be configured, but the `copy` parameter
+    # is deprecated and `fs.copyfile` doesn't work with files that are not in
+    # the source directory
+    configuration: cfg,
```
Comment (ermshiperete) [inline core/tests/unit/kmx/meson.build:145]: Strange thing is that `fs.copyfile` works in `core/tests/unit/ldml/keyboards/meson.build` where we also use files from outside of the current meson project. I tried but couldn't get it to work here. This is a hack I don't like, but at least it works.

**Review by rc-swag — APPROVED** [review]
lgtm
haven't really done much with meson outside the tests in core.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.159-alpha](https://keyman.com/downloads/releases/alpha/18.0.159)

File: core/tests/unit/kmx/meson.build (hunk 1 above)
Comment (mcdurdin) [reply core/tests/unit/kmx/meson.build:145]: FWIW: 

https://github.com/mcdurdin/meson/commit/73d07491543fd166460aaa0b9134b3956297e029

**Pull request merged**